Compilation error

**Describe the bug**
Hi there,
Your module requires github.com/golang-sql/sqlexp
This module had no go.mod file until now, and one was added today (https://github.com/golang-sql/sqlexp/blob/master/go.mod) for go 1.16.
After running `go get -u ./...` on my module (which requires yours) I saw:
```
$ go get -u ./...
go: upgraded github.com/golang-sql/sqlexp v0.0.0-20170517235910-f1bb20e5a188 => v0.0.0-20220518152126-d42ba59f5c11
...
```
Trying to build my module, I get the following compilation error:
```
# github.com/denisenkom/go-mssqldb
../../../../pkg/mod/github.com/denisenkom/go-mssqldb@v0.12.0/token.go:790:79: cannot use info.Message (variable of type string) as type fmt.Stringer in struct literal:
	string does not implement fmt.Stringer (missing String method)

Compilation finished with exit code 2
```

**To Reproduce**
See above.
**Expected behavior**
Code that compiles. Thank you!

### Further technical details
Operating system: Ubuntu 20.04.4
Golang version: go1.18.2 linux/amd64

**Additional context**
N/A

--- a/error.go
+++ b/error.go
@@ -14,7 +14,7 @@ type Error struct {
 	Number     int32
 	State      uint8
 	Class      uint8
-	Message    string
+	Message    Message
 	ServerName string
 	ProcName   string
 	LineNo     int32
@@ -23,8 +23,14 @@ type Error struct {
 	All []Error
 }
 
+type Message string
+
 func (e Error) Error() string {
-	return "mssql: " + e.Message
+	return "mssql: " + e.Message.String()
+}
+
+func (m Message) String() string {
+	return string(m)
 }
 
 // SQLErrorNumber returns the SQL Server error number.
@@ -41,7 +47,7 @@ func (e Error) SQLErrorClass() uint8 {
 }
 
 func (e Error) SQLErrorMessage() string {
-	return e.Message
+	return e.Message.String()
 }
 
 func (e Error) SQLErrorServerName() string {

--- a/queries_go19_test.go
+++ b/queries_go19_test.go
@@ -1125,10 +1125,10 @@ func TestMessageQueue(t *testing.T) {
 	active := true
 
 	msgs := []interface{}{
-		sqlexp.MsgNotice{Message: "msg1"},
+		sqlexp.MsgNotice{Message: Message("msg1")},
 		sqlexp.MsgNext{},
 		sqlexp.MsgRowsAffected{Count: 1},
-		sqlexp.MsgNotice{Message: "msg2"},
+		sqlexp.MsgNotice{Message: Message("msg2")},
 		sqlexp.MsgNextResultSet{},
 	}
 	i := 0

--- a/token.go
+++ b/token.go
@@ -601,7 +601,7 @@ func parseError72(r *tdsBuffer) (res Error) {
 	res.Number = r.int32()
 	res.State = r.byte()
 	res.Class = r.byte()
-	res.Message = r.UsVarChar()
+	res.Message = Message(r.UsVarChar())
 	res.ServerName = r.BVarChar()
 	res.ProcName = r.BVarChar()
 	res.LineNo = r.int32()
@@ -615,7 +615,7 @@ func parseInfo(r *tdsBuffer) (res Error) {
 	res.Number = r.int32()
 	res.State = r.byte()
 	res.Class = r.byte()
-	res.Message = r.UsVarChar()
+	res.Message = Message(r.UsVarChar())
 	res.ServerName = r.BVarChar()
 	res.ProcName = r.BVarChar()
 	res.LineNo = r.int32()
@@ -773,7 +773,7 @@ func processSingleResponse(ctx context.Context, sess *tdsSession, ch chan tokenS
 			}
 			errs = append(errs, err)
 			if sess.logFlags&logErrors != 0 {
-				sess.logger.Log(ctx, msdsn.LogErrors, err.Message)
+				sess.logger.Log(ctx, msdsn.LogErrors, err.Message.String())
 			}
 			if outs.msgq != nil {
 				_ = sqlexp.ReturnMessageEnqueue(ctx, outs.msgq, sqlexp.MsgError{Error: err})
@@ -784,7 +784,7 @@ func processSingleResponse(ctx context.Context, sess *tdsSession, ch chan tokenS
 				sess.logger.Log(ctx, msdsn.LogDebug, fmt.Sprintf("got INFO %d %s", info.Number, info.Message))
 			}
 			if sess.logFlags&logMessages != 0 {
-				sess.logger.Log(ctx, msdsn.LogMessages, info.Message)
+				sess.logger.Log(ctx, msdsn.LogMessages, info.Message.String())
 			}
 			if outs.msgq != nil {
 				_ = sqlexp.ReturnMessageEnqueue(ctx, outs.msgq, sqlexp.MsgNotice{Message: info.Message})

